pkg/controller.v2/controller.go

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Should we name this file `setup.go` to make it more descriptive ?

Reply:
Sure.

Reply:
Done.